src/codegen/codegen_call.rs

```diff
@@ -101,29 +153,36 @@ pub fn codegen_call<'a>(
             let operand1_name = get_var_name(
                 &instruction.get_operand(0).unwrap().left().unwrap(),
                 solver,
-                namespace
+                namespace,
             );
             let operand2_name = get_var_name(
                 &instruction.get_operand(1).unwrap().left().unwrap(),
                 solver,
-                namespace
+                namespace,
             );
 
             let lvalue_var_name_1 = format!("{}.0", get_var_name(&instruction, solver, namespace));
             let lvalue_var_1 = Int::new_const(solver.get_context(), lvalue_var_name_1);
-            let rvalue_var_1 = Int::add(solver.get_context(), &[
-                &Int::new_const(solver.get_context(), operand1_name),
-                &Int::new_const(solver.get_context(), operand2_name)
-            ]);
-            
+            let rvalue_var_1 = Int::add(
+                solver.get_context(),
+                &[
+                    &Int::new_const(solver.get_context(), operand1_name),
+                    &Int::new_const(solver.get_context(), operand2_name),
+                ],
+            );
+
             let assignment_1 = lvalue_var_1._eq(&rvalue_var_1);
 
             let lvalue_var_name_2 = format!("{}.1", get_var_name(&instruction, solver, namespace));
             let min_int = Int::from_i64(solver.get_context(), min_int_val);
             let max_int = Int::from_i64(solver.get_context(), max_int_val);
-            let rvalue_var_2 = Bool::or(solver.get_context(), &[&rvalue_var_1.gt(&max_int), &rvalue_var_1.lt(&min_int)]);
-            
-            let assignment_2 = Bool::new_const(solver.get_context(), lvalue_var_name_2)._eq(&rvalue_var_2);
+            let rvalue_var_2 = Bool::or(
+                solver.get_context(),
+                &[&rvalue_var_1.gt(&max_int), &rvalue_var_1.lt(&min_int)],
+            );
+
+            let assignment_2 =
+                Bool::new_const(solver.get_context(), lvalue_var_name_2)._eq(&rvalue_var_2);
```

Comment:
I kind of dislike these kind of changes unless it is for like chained function calls/function args or overly long lines. Do you know what the default line length is that `fmt` uses? If it isn't too long and easy to change, maybe increase it? (120 doesn't seem too long)

Reply:
Adjusted to 160 characters so most lines are not wrapped by `cargo fmt`.